README.md

```diff
@@ -287,7 +287,9 @@ RULE: We have seen requests that were mostly based on old "I said supported (a
 RULE: weakly defined term to begin with) in a contract, so it has to be in main"
 RULE: feelings, but with sometimes no true reason - neither technically nor
 RULE: helping the user base of Ubuntu. Hence we need to ask for that clearly.
-TODO: - The binary package TBD needs to be in main to achieve TBD
+TODO-A: - The binary packages TBD needs to be in main to achieve TBD
```

Comment:
```suggestion
TODO: - The binary packages TBD needs to be in main to achieve TBD
```

This first line is just TODO, only the next is "A or B".
Other than that +1 on this - thanks for providing the PR